app/src/components/adminConsole/currentCup/CreateCupButton.tsx

```diff
@@ -31,11 +41,12 @@ const CreateCupButton: React.FunctionComponent = (props: {
             isOpen: false,
             timeCreated: new Date(),
             name: newCupName,
+            timePublished: new Date(),
```

Comment:
This is incorrect isn't it? The `timePublished` field should probably be nullable.

Reply:
done - it was already nullable, just hadn't updated it here